### PR TITLE
Adds coroner access

### PR DESCRIPTION
**Pull request by Vivalas**
Adds coroner access. Coroner starts with coroner access and it is used to access the coroner's little office in the morgue.

Doctors and other crew can still visit the morgue, but the little autopsy and office area is off limits to everyone except coroner, detective, and CMO. No longer must the coroner fight a constant battle for privacy among the crew.

:cl:
add: Coroner's office is now off limits to doctors and other crew.
/:cl:

### CODE REVIEW

**Comment by Kyep** [comment]
Add access_coroner to /proc/get_all_accesses()

**Comment by Vivalas** [comment]
Gonna wait for approval before I fix whatever the hell travis is yelling at me for.

**Comment by tigercat2000** [comment]
Travis is just broken at the moment

**Comment by taukausanake** [comment]
I wonder why the Coroner didn't have their own access area to start with

**Comment by SamHPurp** [comment]
@taukausanake Because I originally coded the Coroner job role, and disagree with the need for it.

Not the end of the world certainly if it was merged, I just don't think it was necessary.

**Comment by Ralta** [comment]
To be honest, I think the CMO should absolutely be able to get in, and I'm not sure how you can possibly defend against that.

As for the doctors and detective, it's kinda part of their job roles but losing the access is kinda whatever because _who actually barges into the morgue office ever?_

**Comment by Jountax** [comment]
From what I understand, this restricts the office to only allow the coroner, CMO, and detective in. the only three people who ever need to be in there, IMO. Ever since the split from coroner being a doctor alt title doing autopsies is no longer a part of doctor's job description, just like making medicine is not part of their job and they have to leave that up to the chemists.

Why should doctors be allowed into the morgue office where the actual autopsy tools are? To perform autopsies? That's the coroner's job. To steal things? That's not part of anyone's job. To annoy the coroner? That's the clown's job.

**Comment by Vivalas** [comment]
@Jountax Detective already had access, so I left it. This PR is focued on restricting access to the morgue office area from the rest of medical. The reason is, and I'm not sure if other coroners have had similar issues, but when I play coroner I tend to have problems with random doctors entering my office area and moving my things around, when they just don't need to be in that area,

**Comment by Ralta** [comment]
You might wanna change the description. It currently states:

"...but the little autopsy and office area is off limits to coroner, detective, and CMO."

I thought I'd misread it, but that basically means 'cannot be accessed by', so presumably you meant "...off limits to everyone _except_ coroner, detective and CMO."

File: code/game/jobs/access.dm [hunk 1]
```diff
@@ -73,6 +73,7 @@
 /var/const/access_minisat = 75
 /var/const/access_mineral_storeroom = 76
 /var/const/access_network = 77
+/var/const/access_coroner = 78
```
Comment (tigercat2000) [inline code/game/jobs/access.dm]: Add this to get_all_accesses()

**Comment by Aurorablade** [comment]
As a coroner player, I don't mind people bringing me bodies as long as they do not wordlessly drop it and leave this also doesen't cover a few bases: What is there is not CMO and/or coroner? Its inefficient to ask command constantly or the ai (though you COULD ask the ai or  borg) and the detective should not be acting a sub coroner.

**Comment by Vivalas** [comment]
@Aurorablade Bodies can be dropped in the morgue and removing detective access is out of the scope of this PR. It was already there to begin with.

Anyways I'll get to fixing this by this weekend, been tied up IRL.

**Comment by tigercat2000** [comment]
Please update and fix conflicts or this will be closed.

**Comment by tigercat2000** [comment]
Feel free to open a new PR with the requested changes.

**Pull request closed without merging**